CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Change Log
+
+# [0.65.0 (November 17th, 2020)](https://github.com/eclipse/eclipse.jdt.ls/milestone/83?closed=1)
+ * other - Improve tracing capability of m2e through m2e.logback.configuration. See [#1589](https://github.com/eclipse/eclipse.jdt.ls/pull/1589).
+ * other - Infer expressions if there is no selection range when extracting method. See [#1585](https://github.com/eclipse/eclipse.jdt.ls/pull/1585).
+
```

Comment:
Add https://github.com/eclipse/eclipse.jdt.ls/pull/1596

Reply:
Done. Also moved expected release to 18th.